### DISCUSSION TITLE
Birdwatcher: showing very less options using show

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#27850] [Q&A and General discussion]
Hi team,
As per documentation birdwatcher having many option in show command but in my deployment its showing very limited options.
Milvus(********) > show --help
Usage:
   show [command]

Available Commands:
  channel-watch      display channel watching info from data coord meta store
  checkpoint         list checkpoint collection vchannels
  collection-history display collection change history
  collection-loaded  display information of loaded collection from querycoord
  collections        list current available collection from RootCoord
  current-version    
  index              
  querycoord-channel display querynode information from querycoord cluster
  querycoord-cluster display querynode information from querycoord cluster
  querycoord-task    display task information from querycoord
  replica            list current replica information from QueryCoord
  segment            display segment information from data coord meta store
  segment-index      display segment index information
  segment-loaded     display segment information from querycoord
  session            list online milvus components

Flags:
  -h, --help   help for show

Use " show [command] --help" for more information about a command.
Milvus(*******) > 

We are using version 1.0.0 as the latest one is not available in Dockerhub.
https://hub.docker.com/r/milvusdb/birdwatcher/tags

**Comment by congqixia** [comment]
there are lots of update since version 1.0.0，it's recommended to use latest release, which is v1.0.3

**Reply by navneetrajput06** [reply]
But new version repo is not available on Docker hub. can you please share the link if its there.
https://hub.docker.com/r/milvusdb/birdwatcher/tags
